dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java

```diff
@@ -34,7 +34,7 @@
     }
 
     public static DssRestContentType fromText(String text) {
-        return textMap.get(text);
+    	return textMap.get(text.split(";")[0]);
```

Comment:
Hi, if text can be null?

Reply:
Oh, I didn't consider that text can be null. I'll fix this. Thank you!